frontend/src/components/Filterbox.js

```diff
@@ -0,0 +1,127 @@
+import {Autocomplete, Button, TextField} from "@mui/material";
+import styled from "styled-components";
+
+export default function FilterBox({setEventFilter}) {
+
+    const cities = [
+        {label: 'Hamburg'},
+        {label: 'Berlin'},
+        {label: 'Stuttgart'},
+    ];
+
+    const genres = [
+        {label: 'Concerts'},
+        {label: 'Sport'},
+        {label: 'Comedy'},
+    ];
+
+    const handleNameChange = (event) => {
+
+        if (event.target.value === "") {
```

Comment:
```suggestion
        setEventFilter({...eventFilter, name})
```

Reply:
saves you many lines of code. the same should work for the other functions as well